NamedPipeWrapper/NamedPipeServer.cs

```diff
@@ -74,13 +86,25 @@ public class Server<TRead, TWrite>
 		private volatile bool _shouldKeepRunning;
 		private volatile bool _isRunning;
 
+		private readonly NamedPipeServerStream _server;
+		private readonly StreamReader _reader;
+
 		/// <summary>
 		/// Constructs a new <c>NamedPipeServer</c> object that listens for client connections on the given <paramref name="pipeName"/>.
 		/// </summary>
 		/// <param name="pipeName">Name of the pipe to listen on</param>
-		public Server(string pipeName)
+		/// /// <param name="serverOnlyFlag">Flag indicating whether we are running this as server-only or not</param>
+		public Server(string pipeName, bool option = false)
 		{
 			_pipeName = pipeName;
+			_option = option;
+
+			if (_option)
+			{
+				_server = new NamedPipeServerStream(pipeName, PipeDirection.InOut, 10, PipeTransmissionMode.Message);
+				_reader = new StreamReader(_server);
```

Comment:
It looks like the `NamedPipeConnection` already supports 2-way communication. Why use the `NamedPipeServerStream`?

Reply:
I managed to get the 2-way communication working by looking at the [ExampleCLI](https://github.com/twosense/named-pipe-wrapper/tree/master/ExampleCLI) and modifying their sample. However, 2-way communication didn't work for me when `_option` was true.

My experiment results:

- When `_option` is **true** and server is started by calling **`Start()`** -> I get "Named pipe is broken or disconnected" error

- When `_option` is **true** and server is started by calling **`StartServerOnly()`** -> No exception, but 2-way communication doesn't work 

- When `_option` is **false** and server is started by calling **`StartServerOnly`** -> `System.NullReferenceException`

- When `_option` is **false** and server is started by calling **`Start`** ->  2-way communication works